lib/screens/v2/widget_instance/common/base_alert.ex

```diff
@@ -9,12 +11,13 @@ defmodule Screens.V2.WidgetInstance.Common.BaseAlert do
   alias Screens.Util
   alias Screens.V2.WidgetInstance.Alert, as: AlertWidget
   alias Screens.V2.WidgetInstance.ReconstructedAlert
+  alias Screens.V2.WidgetInstance.DupAlert
 
   @type stop_id :: String.t()
 
   @type route_id :: String.t()
 
-  @spec home_stop_id(AlertWidget.t() | ReconstructedAlert.t()) :: String.t()
+  @spec home_stop_id(AlertWidget.t() | ReconstructedAlert.t() | DupAlert.t()) :: String.t()
```

Comment:
Line 24 needs to include `DUP`

Reply:
Pushing my changes related to this shortly--all of the values that BaseAlert depends on have been laid out in a new `SingleAlertWidget` protocol now, so these functions all just operate on `SingleAlertWidget.t()` instead of the union of `AlertWidget.t()`, `ReconstructedAlert.t()`, `DupAlert.t()`.